commands/audit/scarunner.go

```diff
@@ -226,11 +225,7 @@ func GetTechDependencyTree(params xrayutils.AuditParams, tech coreutils.Technolo
 	case coreutils.Go:
 		fullDependencyTrees, uniqueDeps, err = _go.BuildDependencyTree(params)
 	case coreutils.Pipenv, coreutils.Pip, coreutils.Poetry:
-		fullDependencyTrees, uniqueDeps, err = python.BuildDependencyTree(&python.AuditPython{
-			Server:              serverDetails,
-			Tool:                pythonutils.PythonTool(tech),
-			RemotePypiRepo:      params.DepsRepo(),
-			PipRequirementsFile: params.PipRequirementsFile()})
+		fullDependencyTrees, uniqueDeps, err = python.BuildDependencyTree(serverDetails, tech, params)
```

Comment:
pass `python.AuditPython` that holds all the needed parameters for this operation.
like at maven, pass it the new parameter:
```go
IsCurationCmd:           params.IsCurationCmd()
```

Reply:
answered here https://github.com/jfrog/jfrog-cli-security/pull/39#discussion_r1542933177

Reply:
Fixed